Make the loader message update every 5 seconds

paulkaplan opened the pull request:
### Resolves

_What Github issue does this resolve (please include link)?_

Resolvers https://github.com/LLK/scratch-gui/issues/1552

### Proposed Changes

_Describe what this Pull Request does_

Rotate the loading messages every 5 seconds to make sure people know it is still going!

### Reason for Changes

_Explain why these changes should be made_

From @thisandagain 
> While loading projects, displaying a single message (e.g. "Loading sprites ...", "Loading costumes ...", etc.) can lead to the perception that the editor has "hung" and gotten stuck. My instinct is that this is reinforced by the behavior of the current Scratch 2.0 loading screen which shows a progress bar.
> My recommendation would be for us to rotate through messages every few seconds to better reinforce that the 3.0 editor is "doing something".


![loading-messages](https://user-images.githubusercontent.com/654102/37711017-d2df85bc-2ce5-11e8-8886-ade1db586a1e.gif)

kyleplo commented on src/components/loader/loader.jsx:
```diff
@@ -5,140 +5,171 @@ import styles from './loader.css';
+    {
+        message: (
+            <FormattedMessage
+                defaultMessage="Preparing emojiis …"
```
Emojis is spelled wrong

kyleplo commented on src/components/loader/loader.jsx:
```diff
@@ -5,140 +5,171 @@ import styles from './loader.css';
+    {
+        message: (
+            <FormattedMessage
+                defaultMessage="Loading extensions …"
```
What if the project doesn't have extensions?

thisandagain commented:
@paulkaplan Looks like the build failed. I'm attempting to re-run, but regardless you may want to resolve the type @kyleplo caught above.

paulkaplan commented:
Woops, linting error was breaking the build. Fixed. Also fixed the typo, thanks @kyleplo. In terms of the extensions line, again this loading is temporary since we do not have the ability to get fine grained loading info right now.

BryceLTaylor commented:
@paulkaplan Could you add some test cases to this issue?
